src/app/app.module.ts

```diff
@@ -37,14 +39,15 @@ import {
     NgxSpinnerModule,
     ToastrModule.forRoot(),
     BrowserAnimationsModule,
+    NgxSpinnerModule
```

Comment:
`NgxSpinnerModule` is already imported on line 39.

Reply:
Thanks, @Victoradukwu, I have worked on this.